grunt_tasks/plugin.js

```diff
@@ -369,16 +370,16 @@ module.exports = function (grunt) {
                                 [`girder_plugins/${plugin}/node`]: pluginNodeDir,
                                 [`girder_plugins/${plugin}`]: webClient
```

Comment:
Thinking another minute on this, perhaps the first `node` alias could be removed, but maybe not the second, which is for importing code from the plugin itself.